README.md

```diff
@@ -1 +1,31 @@
-# Enumerable_project
\ No newline at end of file
+# Enumerable_project
```

Comment:
[OPTIONAL]
Consider using [this template](https://github.com/microverseinc/readme-template/blob/master/README.md) for your README files.